gnovm/pkg/gnolang/debugger.go

```diff
@@ -257,17 +257,21 @@ func debugUpdateLocation(m *Machine) {
 	for i := nx - 1; i >= 0; i-- {
 		expr := m.Exprs[i]
 		if l := expr.GetLine(); l > 0 {
-			m.Debugger.loc.Line = l
-			m.Debugger.loc.Column = expr.GetColumn()
+			if col := expr.GetColumn(); col > 0 {
```

Comment:
seems strange, is this needed?

Reply:
this is an effort to fix incorrect line column, @mvertes , would you please take a look at this?